Assignment2/FORGIS98_fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,66 @@
+{
+  "@context" : {
+    "base" : "http://example.org/",
+    "data" : "http://example.org/data",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "includes" : "rdfs:label",
+    "hasMeasurement" : "rdfs:label",
+    "hasOwner" : "rdfs:label",
+    "hasTemperature" : "rdfs:label",
+    "atTime" : "rdfs:label",
+    "hasName" : "rdfs:label",
```

Comment:
Check my comments on attribute definitions that I wrote on your JSONtoJSON-LD file

Reply:
These changes are still missing.